### DISCUSSION TITLE
Message "Ack type"

### DISCUSSION THREAD

**Discussion by aleverdes** [MirageNet/Mirage#1160] [Q&A] [answered]
Gentlemen, thank you for such a wonderful framework.

I have a problem: in most cases, messages do not reach the joined players (and vice versa) even in the case of localhost. Messages are trying to be sent, it is in the logs, but the main difference from the "working" case is that a message about a certain "Ack type" is spammed. What could be wrong?

**Comment by James-Frowen** [comment, marked as answer]
I'm not sure what the issue could be.

The Ack packets should only be sent if no other packets were recently sent. I dont remember if they are sent right at the start too (after connecting), or if they wait to receive a packet first, but that shouldn't negatively effects things.

They should only be sent after connecting, so it is odd that it would connect, and then break. 

Try checking what is being received/sent by the socket class, that should narrow down what part is breaking

**Reply by aleverdes** [reply]
It was my bad. I did not recreate the message handler when the scene was restarted (due to static and disabled reloading of the app domain). I rewrote the algorithm for its initialization and the multiplayer started working like clockwork. Thank you!
